tests/docs/smoke-all/typst/custom-fonts.qmd

```diff
@@ -0,0 +1,16 @@
+---
+title: font typst
+format: 
+  typst:
+      mainfont: "Agbalumo"
+      font-paths: myfonts
+#TODO: How to test the PDF for correct font usage ?
```

Comment:
Just FWIW, I don't know either :(

I think we should spend some time in 1.5 figuring out how to test the content of PDFs more explicitly. That would be useful for all of our PDF output generation.